.github/workflows/pypi-publish.yml

```diff
@@ -56,15 +56,15 @@ jobs:
           pypi_token: ${{ secrets.LANCEDB_PYPI_API_TOKEN }}
           fury_token: ${{ secrets.FURY_TOKEN }}
   mac:
-    timeout-minutes: 60
+    timeout-minutes: 90
     runs-on: ${{ matrix.config.runner }}
     strategy:
       matrix:
         config:
           - target: x86_64-apple-darwin
             runner: macos-13
```

Comment:
warp does not support x86-64 for mac, so we have to stick with the default runner...